### PR TITLE
Bugfix #107

### PR DESCRIPTION
**Pull request by svenauhagen**
Hi,

this is a bugfix for #107 

Thanks
Sven

### CODE REVIEW

**Comment by audiolion** [comment]
Can you write a test case for passing a naive datetime and asserting the outcome?

**Comment by svenauhagen** [comment]
Yes, I will write a test later today and submit it too

File: src/auditlog_tests/test_settings.py [hunk 1]
```diff
@@ -30,14 +30,6 @@
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': 'auditlog_tests.db',
     },
-    'postgres': {
```
Comment (audiolion) [inline src/auditlog_tests/test_settings.py]: why did you remove the postgres database tests? We have some tests that test postgres functionality specifically, I can understand that when you ran locally you might not have had a postgres server / db setup, but we shouldn't be removing it

Comment (svenauhagen) [reply src/auditlog_tests/test_settings.py]: Sorry, I did not see that I committed that file as well.
I removed it for local testing on my end.
Do you want me to submit a complete new PR?

**Comment by svenauhagen** [comment]
Sorry for the wrong commit I rebased and it should be fine now

**Comment by audiolion** [comment]
yeah that is totally fine, thanks for rebasing, you didn't have to, I squash the commits going in so it wouldn't matter too much, but better to do it this way all the same for clarity with git's history.

**Comment by svenauhagen** [comment]
Hi,

did this fix get merged at all?

Best
Sven

**Comment by audiolion** [comment]
sorry lost track of this, working on merging now

**Pull request merged**